lib/elixir/lib/kernel.ex

```diff
@@ -1126,6 +1126,27 @@ defmodule Kernel do
     :erlang.spawn_monitor(module, fun, args)
   end
 
+  @doc """
+  Applies a function over a value returns the initial value.
+
+  Useful for running synchronous side effects when you don't want their
+  result to affect a pipe chain.
+
+  ## Examples
+
+    iex> tap(1, fn x -> x + 1 end)
+    1
+
+    iex> tap(%{a: 1}, &IO.inspect(Map.keys(&1)))
+    %{a: 1}
```

Comment:
This doctest prints during tests. I want a side-effect that doesn't print but could be useful for the reader. Nothing comes in mind.

Reply:
How about a `send/2` to somewhere?